lstchain/calib/camera/calibrator.py

```diff
@@ -38,6 +39,12 @@ class LSTCameraCalibrator(CameraCalibrator):
         help='Path to drs4 time calibration file'
     ).tag(config=True)
 
+    time_sampling_correction_path = Unicode(
```

Comment:
Use `ctapipe.core.traits.Path` for anything that is a Path. You can use it to make sure that the path actually exista and is a file:

` = Path(exists=True, directory_ok=False)`

Reply:
Done